extensions/amp-inputmask/0.1/text-mask.js

```diff
@@ -56,7 +67,12 @@ export class TextMask {
     this.controller_.mask();
 
     Services.formSubmitPromiseForDoc(element).then(formSubmitService => {
-      formSubmitService.beforeSubmit(e => this.handleBeforeSubmit_(e.form));
+      formSubmitService.beforeSubmit(e => {
```

Comment:
nit: since you're only using `form` from `e`, you could destructure here: `({form}) => {`

Reply:
Good idea, will add in follow-up PR.